omod/src/test/resources/TagResourceDataSet.xml

```diff
@@ -0,0 +1,29 @@
+<dataset>
```

Comment:
its the same dataset, but its not possible to access a file in the api from omod/src/test since the project structure is such. plus having different datasets might be useful, since the api, and rest+ui have a few different features.